app/controllers/quby/answers_controller.rb

```diff
@@ -240,5 +239,13 @@ def default_to_textvar_values(answer)
         end
       end
     end
+
+    # I18n.t based on the questionnaire's configuration.
+    # This is done instead of configuring I18n.locale to allow for example
+    # English questionnaires within a Dutch RoQua.
+    def translate(key, options = {})
```

Comment:
Op zich hoeft dit niet, een controller action wordt altijd op zichzelf aangeroepen en , dus als je voor de actie gewoon I18n op de taal van de questionnaire zet heeft roqua daar nooit last van.

dus in een before filter:

I18n.locale = @answer.questionnaire.language